apps/fluent-tester/src/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -435,7 +435,7 @@ export const CalloutTest: React.FunctionComponent = () => {
     win32Status: 'Beta',
     uwpStatus: 'Backlog',
     iosStatus: 'Backlog',
-    macosStatus: 'Backlog',
+    macosStatus: 'Beta',
```

Comment:
I've been erring for Beta for all components that are under the old compose framework, partially because we will rewrite it in the future and partially for consistency. But if you're ok w/ calling this component production for macOS then I'm fine with that.

Reply:
Ah, with that context, everything else makes sense. I guess I'm OK with that.